src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/SqlAction.cs

```diff
@@ -58,7 +58,7 @@ TraceId traceId
                 {
                     await using NpgsqlCommand command =
                         new(
-                            $@"SELECT EdfiDoc FROM dms.Document WHERE DocumentPartitionKey = $1 AND DocumentUuid = $2 AND ResourceName = $3 {SqlFor(lockOption)};",
+                            $@"SELECT * FROM dms.Document WHERE DocumentPartitionKey = $1 AND DocumentUuid = $2 AND ResourceName = $3 {SqlFor(lockOption)};",
```

Comment:
You never know what big metadata columns might be added to the `Document` table later. Let's just select `EdfiDoc` and the LastModifieds to minimize data transfer/conversion and future surprises. Either create a new simple `record` type for the return, or just return a (`JsonNode`, `DateTime`, `TraceId`) tuple.